src/constants.ts

```diff
@@ -16,7 +16,7 @@ export const chunkTypeOther = 'other';
  * where the not matched imports should be placed
  */
 export const THIRD_PARTY_MODULES_SPECIAL_WORD = '<THIRD_PARTY_MODULES>';
-export const BUILTIN_MODULES = builtinModules.join('|');
+export const BUILTIN_MODULES = `^(?:${builtinModules.join('|')})$`;
```

Comment:
Careful, this is missing support for node protocol imports, e.g. `require("node:fs")`

```js
`^(${builtinModules.join("|")}|${builtinModules.map(module => 'node:'+module).join("|")})$`
```

Reply:
Or perhaps a bit simpler:

```js
`^(node:)?(${builtinModules.join("|")})$`
```

Reply:
I should have run all the tests before pushing it up.  :)

I tweaked the regex a bit to allow an optional `node:` prefix.

Reply:
You said you tweaked the regex @IanVS, but I admit, I don't understand how that tweak works? -- or the tweak hasn't been pushed up yet?

Here's my playground: https://regex101.com/r/d7kjI0/1

Reply:
(I see the fix now)